AngularApp/src/app/modules/dashboard/dashboard.module.ts

```diff
@@ -70,6 +71,11 @@ export const DashboardModuleRoutes: ModuleWithProviders = RouterModule.forChild(
             path: 'edit',
             component: TabDevelopComponent
           }
+          ,
+          {
+            path: 'datasource',
+            component: TabDataComponent
```

Comment:
Once you create a new component ('ng generate component [name]'), update this too.